hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/transfer/TransferContextImpl.java

```diff
@@ -186,6 +190,21 @@ public void validateHbarAllowances() {
                         accountStore.get(aa.accountIDOrElse(AccountID.DEFAULT)), topLevelPayer, aa.amount());
             }
         }
+        final var nftStore = context.readableStore(ReadableNftStore.class);
```

Comment:
Why do we need to validate only Hbar and NFT allowances here (not token transfers)🤔 ?

Reply:
Confirmed that these should be deleted after differential testing. It is just to match the mono-service responses. 

A FUTURE removal comment will be added in later PR.